x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.tsx

```diff
@@ -25,14 +25,14 @@ import { EnginesListLogic } from './engines_list_logic';
 export const EnginesList: React.FC = () => {
   const { fetchEngines, onPaginate } = useActions(EnginesListLogic);
   const { meta, results } = useValues(EnginesListLogic);
-  const [searchQuery, setSearchValue] = useState('');
+  // const [searchQuery, setSearchValue] = useState('');
```

Comment:
I think I'd prefer to leave this in place and have it not work until the backend supports `?search_query` rather than committing commented out code.